pkg/restic/exec_commands.go

```diff
@@ -153,6 +155,9 @@ func decodeBackupStatusLine(lastLine []byte) (backupStatusLine, error) {
 // have a newline at the end of it, so this returns the substring between the
 // last two newlines.
 func getLastLine(b []byte) []byte {
+	if b == nil || len(b) == 0 {
```

Comment:
you should be able to just do `if len(b) == 0 {` since a nil slice also has a length of zero. Not blocking.